src/main/java/mate/academy/service/ShoppingCartService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.service;
+
+import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+
+public interface ShoppingCartService {
+    /**
+     * This method is responsible for adding a Ticket to the ShoppingCart
+     * @param movieSession contains the information required for the ticket
```

Comment:
Let's look for the JavaDocs in [Object](https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html) class. 
JavaDocs here begins with a verb in the present tense. Let's do the same.
 
It would be great to use `@link`. [Link](https://www.baeldung.com/javadoc-see-vs-link#link).

For example:
```suggestion
     * Adds a {@link Ticket} to the ShoppingCart
     * @param movieSession - {@link MovieSession} that contains the information required for the {@link Ticket}.
```